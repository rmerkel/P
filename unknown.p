var x;
begin
	x # 123;	{	unknown operator "#"!
end .
