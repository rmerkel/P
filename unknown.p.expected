./pl0c: Unknown token: '#, (0x23) near line 3
./pl0c: identifier is not a variable or a procedure 'x' near line 3
./pl0c: expected 'end' got 'Number' near line 3
./pl0c: expected '.' got 'Number' near line 3
# unknown.p, 2: var x;
    0: call 0, 2
    1: halt
# unknown.p, 3: begin
    2: enter 1
# unknown.p, 4: 	x # 123;
    3: ret
#unknown.p, 4: end .

